integration-tests/README.md

```diff
@@ -181,11 +183,15 @@ test methods -  testDomainOnPVUsingWLST, testDomainOnPVUsingWDT, testTwoDomainsM
 
 Utility classes:
 
-Operator - contains methods to create/destroy operator, verify operator created, scale using rest api, etc
-Domain - contains methods to create/destroy domain, verify domain created,deploy webapp, load balancing, etc
-PersistentVolume - to create PV
-LoadBalancer - to create load balancer
-Secret - to create secret
+Operator - constructor takes yaml file with operator properties and generates operator valus yaml with required properties and certs,  creates service account, namespace and calls helm install using the generated values yaml file. Also contains methods to delete operator release, verify operator created and ready, scale using rest api, verify a given domain via rest, verify external rest service, etc  <p>
+Domain - constructor takes Map with domain, LB, PV properties and creates domain crd, LB operator/ingress and PV artifacts using the sample scripts provided in the project. Also contains helper methods to destroy domain by deleting domain crd, verify domain created and servers are ready, deploy webapp, verify load balancing of http requests, etc <p>
+PersistentVolume - runs k8s job to create PV directory and creates PV and PVC using sample scripts  <p>
+LoadBalancer - creates load balancer, currently TREFIK and VOYAGER are supported <p>
+Secret - creates a k8s secret <p>
+TestUtils - mostly runs kubectl commands. Contains utility methods to check if a pod is created, ready, deleted, service created, get pod restart cnt, get cluster replica, delete PVC, check PV released, create rbac policy, create wldf module, etc. <p>
+ExecCommand - Class for executing shell commands from java <p>
+ExecResult - Class that holds the results of using java to exec a command (i.e. exit value, stdout and stderr) <p>
+K8sTestUtils - uses k8s java client api, this is used only for delete domain use cases for now. <p>
 
 # How to run the Java integration tests
 
```

Comment:
Please update these instructions to look something like:
* Maven and latest Git should be in PATH
* export JAVA_HOME
* export WEBLOGIC_IMAGE_NAME and WEBLOGIC_IMAGE_TAG if different from `store/oracle/weblogic` and `12.2.1.3`.
* Setup docker access to WebLogic 12c Images
  * Method 1
    * Setup a personal account on hub.docker.com
    * Then sign in to hub.docker.com and signup for access to WebLogic 12c Images via https://hub.docker.com/_/oracle-weblogic-server-12c
    * Then export the following before running the tests:
      * export DOCKER_USERNAME=<docker user>
      * export DOCKER_PASSWORD=<docker pass>
      * export DOCKER_EMAIL=<docker email>
  * Method 2
      * Make sure the weblogic image i.e. store/oracle/weblogic:12.2.1.3 already exists locally
      * Make sure the weblogic image has patch p29135930 (required for the WebLogic Kubernetes Operator).
        * If not, see [https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-patch-wls-for-k8s].

* Command to run the tests: ```
 mvn clean verify -P java-integration-tests 2>&1 | tee log.txt
 ```

Reply:
Done